README.md

```diff
@@ -469,6 +470,10 @@ Specifies the Redis server host used by the build status codes server (see `ENAB
 
 Specifies the Redis server port used by the build status codes server (see `ENABLE_BSC_CHANNEL`). 
 
+## Running images in emulator
+
+It's possible to run the images built by Pieman in QEMU. The nearby project [MMB](https://github.com/tolstoyevsky/mmb) simplifies the process. The project is the set of Dockerfiles and assets for building Docker images with different services. Now [QEMU](https://github.com/tolstoyevsky/mmb/tree/master/qemu), which is one of the services, helps running the images based on Ubuntu 18.04 «Bionic Beaver» (64-bit) for Raspberry Pi.
+
```

Comment:
Maybe it has the sense to specify the Raspberry model here as you made in issue reply?